src/vaults/vaults.utils.ts

```diff
@@ -237,32 +228,39 @@ export async function loadVaultEventPerformances(
   return estimateVaultPerformance(chain, vaultDefinition, data);
 }
 
+/**
+ * Extrapolates a one year APR for a given vault based on compounding and emissions based on $100 deposit.
+ * @param compoundApr Base compound APR of vault
+ * @param emissionApr Emission APR of the emitted vault
+ * @param emissionCompoundApr Derivative compound APR of the tmitted vault
+ * @returns Extraposedat one year APR given current yields continue
+ */
 export function estimateDerivativeEmission(
   compoundApr: number,
   emissionApr: number,
   emissionCompoundApr: number,
 ): number {
+  // start with $100 deposited into the vault
   let currentValueCompounded = 100;
   let currentValueEmitted = 0;
   let currentValueEmittedCompounded = 0;
   for (let i = 0; i < 365; i++) {
+    // accrue emitted yield from emissionApr
     const emitted = currentValueCompounded * (emissionApr / 365);
     currentValueCompounded += currentValueCompounded * (compoundApr / 365);
+    // accrue compounded yield from emitted tokens
     const emittedCompounded = currentValueEmitted * (emissionCompoundApr / 365);
+    // account for total yield emitted
     currentValueEmitted += emitted;
+    // account for the actual compounding portion on the emitted yield (what we are looking for)
     currentValueEmittedCompounded += emittedCompounded;
+    // account for the compounded yield
     currentValueEmitted += emittedCompounded;
   }
-  const total = currentValueCompounded + currentValueEmitted + currentValueEmittedCompounded;
+  const total = currentValueCompounded + currentValueEmitted;
```

Comment:
need eyes on this one. i think this is correct because `currentValueEmitted` includes `emittedCompounded`